staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -302,6 +315,25 @@ func UpdateInflightRequestMetrics(phase string, nonmutating, mutating int) {
 	}
 }
 
+func RecordFilterLatency(req *http.Request, requestInfo *request.RequestInfo, name string, elapsed time.Duration) {
+	if requestInfo == nil {
+		requestInfo = &request.RequestInfo{Verb: req.Method, Path: req.URL.Path}
+	}
+	scope := CleanScope(requestInfo)
+	verb := canonicalVerb(strings.ToUpper(req.Method), scope)
+	// set verbs to a bounded set of known and expected verbs
+	if !validRequestMethods.Has(verb) {
+		verb = OtherRequestMethod
+	}
+
+	ms := elapsed.Milliseconds()
+	if requestInfo.IsResourceRequest {
+		requestFilterLatency.WithLabelValues(name, cleanVerb(verb, req), requestInfo.APIGroup, requestInfo.APIVersion, requestInfo.Resource).Observe(float64(ms))
```

Comment:
Oh it's ms? I think metrics are usually microseconds or seconds, at least I didn't expect this.